### PR TITLE
GH-3133 / Add WEB3AUTH feature flag to chain info

### PR DESCRIPTION
**Pull request by biafra23**
Handles #3133

Changes proposed in this pull request:
- [x] Add the feature to staging config service to Görli chain (Not in this PR. See: https://safe-config.staging.5afe.dev/admin/chains/feature/15/change/)
- [x] Parse the feature from the config service response and add it to chain info

### CODE REVIEW

File: Multisig/UI/UI Library/View Controllers/InstructionsViewController/InstructionsViewController.swift [hunk 1]
```diff
@@ -42,9 +42,15 @@ class InstructionsViewController: UIViewController, UITableViewDelegate, UITable
     }
     
     @IBAction func didTapButton(_ sender: Any) {
-        let createSafeVC = CreateSafeViewController()
-        createSafeVC.onClose = onClose
-        show(createSafeVC, sender: self)
+
+        if Chain.all.filter({ $0.isSupported(feature: .web3authCreateSafe) }).count > 1 {
```
Comment (MouazAlzahabi) [inline Multisig/UI/UI Library/View Controllers/InstructionsViewController/InstructionsViewController.swift]: do we want to limit network selection as first step just for the case when web3authCreateSafe enabled ?, why not always select it first?
if so, I think the if and else cases should be switched right?

File: Multisig/UI/Safe Management/Create Safe/Setup Safe/CreateSafeSelectNetworkViewController.swift [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+//  CreateSafeSelectNetworkViewController.swift
+//  Multisig
+//
+//  Created by Dirk Jäckel on 14.06.23.
+//  Copyright © 2023 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class CreateSafeSelectNetworkViewController: UIViewController {
```
Comment (MouazAlzahabi) [inline Multisig/UI/Safe Management/Create Safe/Setup Safe/CreateSafeSelectNetworkViewController.swift]: Maybe we can use the SelectNetworkView controller after we alter it ?

File: Multisig/UI/UI Library/View Controllers/InstructionsViewController/InstructionsViewController.swift (hunk 1 above)
Comment (biafra23) [reply Multisig/UI/UI Library/View Controllers/InstructionsViewController/InstructionsViewController.swift]: I asked @TanyaEfremova, and the old flow should be used if there is no network with web3auth enabled.

File: Multisig/UI/Safe Management/Create Safe/Setup Safe/CreateSafeSelectNetworkViewController.swift (hunk 2 above)
Comment (biafra23) [reply Multisig/UI/Safe Management/Create Safe/Setup Safe/CreateSafeSelectNetworkViewController.swift]: If possible, that can be done later, right?

File: Multisig/UI/UI Library/View Controllers/InstructionsViewController/InstructionsViewController.swift (hunk 1 above)
Comment (MouazAlzahabi) [reply Multisig/UI/UI Library/View Controllers/InstructionsViewController/InstructionsViewController.swift]: I have checked the flow, we don't present the InstructionsCiewController with the chains that supports this so we won't reach this point with chains that supports web2 style, so this is not the correct place for this check

Comment (biafra23) [reply Multisig/UI/UI Library/View Controllers/InstructionsViewController/InstructionsViewController.swift]: You are right. I misunderstood your first comment. Will fix it.

**Review by MouazAlzahabi — APPROVED** [review]

**Review by MouazAlzahabi — APPROVED** [review]

**Pull request merged**